.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ concurrency:
 jobs:
   formatting-linting:
     name: Formatting, linting & changeset checks
-    runs-on: ${{ vars.RUNNER_LARGE || 'ubuntu-latest-l' }}
+    runs-on: ${{ vars.RUNNER_LARGE || 'blacksmith-16vcpu-ubuntu-2204' }}
```

Comment:
This is gonna need a logic change within our variables in order to actually work here, just a note